Account settings

wshi01 opened the pull request:
There is also a bug that when you click on the top left of the screen you will go into the account settings, after you entered and exited the account settings page using the back button. There must be a navigation pop bug I need to fix.

wshi01 committed:
added base template for the hamburger menu

wshi01 committed:
Merge branch 'hMRZQ21:main' into main

wshi01 committed:
Account template
Added account template with some colors
One button on the hamburger menu works
There is still no animation
Added a back button on Camera page
On the mainpage I added the Hero widget but animations need to be added later, there might be conflicts with bottom navbar and need to change
     - navigateBottomNavBar function
Bug right now is when new page is opened using navbar the navbar disappears, as evident when clicking on the account settings page
     - current solution is to add the back button for now to access the homepage

wshi01 committed:
Merge branch 'hMRZQ21:main' into main

hMRZQ21 commented:
make sure you work out the kinks, and once it all runs smoothly, we can merge it

wshi01 committed:
Fixes
1. Removed back button: now page navigation relys on system swipe to the right from the left side of screen.
2. Added Cupertino to the tab bar and the page routers to have more iSO animations.
3. Cleaned up some imports

Things that needs fixing:
1. Dynamic sizing of pages edited
2. Choose whether we want Icons pack or Cupertino Icons
